packages/components/src/components/action-pad/action-pad.browser.e2e.tsx

```diff
@@ -98,4 +103,100 @@ describe("calcite-action-pad", () => {
       "calcite-action-group",
     );
   });
+
+  describe("keyboard navigation and selection-mode", () => {
+    it("supports ARIA keyboard navigation and focus management", async () => {
+      const { el } = await mount<"calcite-action-pad">(
+        <calcite-action-pad
+          layout="horizontal"
+          overflow-actions-disabled
+          selection-appearance="neutral"
+        >
+          <calcite-action-group selection-mode="single-persist">
+            <calcite-action appearance="solid" icon="plus" id="action-1" scale="m" text="Add" />
+            <calcite-action appearance="solid" icon="save" id="action-2" scale="m" text="Save" />
+            <calcite-action appearance="solid" icon="trash" id="action-3" scale="m" text="Delete" />
+          </calcite-action-group>
+        </calcite-action-pad>,
+      );
+
+      const action1 = el.querySelector("#action-1") as Action["el"];
+      const action2 = el.querySelector("#action-2") as Action["el"];
+      const action3 = el.querySelector("#action-3") as Action["el"];
+
+      await userEvent.click(action1);
+      expect(document.activeElement).toBe(action1);
+
+      await userEvent.keyboard("{ArrowRight}");
+      expect(document.activeElement).toBe(action2);
+
+      await userEvent.keyboard("{ArrowLeft}");
+      expect(document.activeElement).toBe(action1);
+
+      await userEvent.keyboard("{End}");
+      expect(document.activeElement).toBe(action3);
+
+      await userEvent.keyboard("{Home}");
+      expect(document.activeElement).toBe(action1);
+    });
+
+    it("single and none(default) selection modes work as expected", async () => {
```

Comment:
Nitpick: `work as expected` is implied, so this could be renamed to `has single and none (default) selection modes`.